pennylane/devices/default_mixed.py

```diff
@@ -112,7 +112,10 @@ class DefaultMixed(QubitDevice):
 
     _reshape = staticmethod(qnp.reshape)
     _flatten = staticmethod(qnp.flatten)
-    _gather = staticmethod(qnp.gather)
+    # Allow for the `axis` keyword argument for integration with broadcasting-enabling
+    # code in QubitDevice. However, it is not used as DefaultMixed does not support broadcasting
+    # pylint: disable=unnecessary-lambda
+    _gather = staticmethod(lambda *args, axis=0, **kwargs: qnp.gather(*args, **kwargs))
```

Comment:
This might be an annoying breaking change to device classes that define their own `_gather` method! What do you think about this?

Reply:
Would it be fine to leave this file unchanged? Or do tests fail?